python/equistore-torch/equistore/torch/operations.py

```diff
@@ -89,7 +89,3 @@ def check_isinstance(obj, ty):
 # with the newly created module
 sys.modules[spec.name] = module
 spec.loader.exec_module(module)
-
-# Step 3: override the objects/functions/classes exposed by this module with the data
-# from the module we just created.
-globals().update(module.__dict__)
```

Comment:
Why is this not necessary anymore?

Reply:
Because we expose all functions to the current namespace with `pec.loader.exec_module(module)`. I think this is leftover when @Luthaf started the work on this.

Reply:
Perfect, thanks for the explanation. I see that it doesn't break anything so all good